Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -575,23 +585,24 @@ open class BasicFormat: SyntaxRewriter {
       trailingTrivia += .space
     }
 
-    var leadingTriviaIndentation = self.currentIndentationLevel
-    var trailingTriviaIndentation = self.currentIndentationLevel
+    var leadingTriviaIndentation: Trivia
+    var trailingTriviaIndentation: Trivia
```

Comment:
I believe since you removed default values in these properties we can change them to `let`'s since they are not mutating anymore.

```suggestion
    let leadingTriviaIndentation: Trivia
    let trailingTriviaIndentation: Trivia
```